meetings/2020-04-23.md

```diff
@@ -0,0 +1,111 @@
+
+## Links
+
+* **Recording**:  https://youtu.be/Da79AzGNiDE
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/852
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* James Snell @jasnell (TSC)
+* Matheus Marchini @mmarchini (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ruben Bridgewater @BridgeAR (TSC)
+* Sam Roberts @sam-github (TSC)
+
+## Agenda
+
+### Announcements
+
+* Myles, Node.js 14 went out this week !   Will be our next LTS release.
+
+### CPC and Board Meeting Updates
+
+* Board meeting this, week.
+  * James asked about DCO side of things.
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* openssl security release 1.1.1g - vulnerability HIGH [#32846](https://github.com/nodejs/node/issues/32846)
+  * Turns out it does not affect Node.js so no need to discussion.
+
+* Nominating @juanarbol as a collaborator [#32621](https://github.com/nodejs/node/issues/32621)
+  * already done, issue closed.
+
+* process: Throw exception on --unhandled-rejections=default [#33021](https://github.com/nodejs/node/pull/33021)
+  * There have been a few different PRs. Today Node.js prints a warning and a
+    deprecation message
+  * There is a strict mode which will through immediately
+  * Ruben -> there were 3 different proposals
+    * Switch default to strict mode. There were objections so that one was closed because
+      tests were failing and different approach might be better.
+   * Change defaults so unhandled rejections hook is higher priority, hook and flag 
+     currently work independently. If you had hook and flag then you would not 
+     get unhandled exception 
+   * 33021 adds “force mode”, through exception if there is no hook in place and is 
+     selected as the new default.
+  * All are/were SemVer major.
+  * James, none of these implement behaviour that I would like to see.
+    * option ok, but not as default.
+  * Anna, deprecation warning is actually different from throwing an exception.
+  * Original plan was to do survey
+  * Matheus: Only people who are affected by the deprecation warning will be broken. With 
+    proposed change handler would be enough so no command line flag required. One thing
+    is that any module can add handler which would cause the override.
+  * Need to integrate discuss in the issue.
+  * Ruben, prefer that we still have survey. 
+  * Ruben will write blog post, idea would be to have survey link at the end. 
+
+* http: align with stream.Writable [31818](https://github.com/nodejs/node/pull/31818)
+  * Rich has asked for TSC concensus or not on overriding the objection. In this comment 
+    https://github.com/nodejs/node/pull/31818#issuecomment-618180372
+
+### nodejs/docker-node
+
+* Suggestion for dropping Yarn from Node 14 release [#1238](https://github.com/nodejs/docker-node/issues/1238)
+  * Missed window, already released, does not need discussion.

```

Comment:
doesn’t this need discussion for v15, at least?

Reply:
> doesn’t this need discussion for v15, at least?

/ping @nschonni What do you think? Too early to discuss that?

Reply:
Probably doesn't hurt. I just mentioned over on the other issue, that there are long standing discussions on other threads about it. If the TSC isn't going to weight it, I don't really want to take another beating from the Yarn folks right now.

PS: There is some WG vs Colloborator divide on the docker-node project that I don't exactly understand, and the push to remove it was more from the Collaborators than anyone on the WG

Reply:
imo the decision to include it initially should have had much broader consensus, beyond the WG, perhaps even the TSC.